### ISSUE TITLE
Does not work for the new JIRA version

### ISSUE THREAD

**Issue by tessi** [bitcrowd/tickety-tick#113]
JIRA changed their CSS identifiers to be obfuscated. (the page looks the same). This results in none of the JIRA pages working with tickety-tick anymore.

<img width="565" alt="screenshot 2018-12-03 at 09 57 30" src="https://user-images.githubusercontent.com/206108/49363463-03b47500-f6e2-11e8-9273-5332cf1ed949.png">

> ☝️ the HTML of a ticket in the backlog-view

We cannot use the (now obfuscated) css classes anymore and must rely on other means of scanning the page. Possible options are probably different per JIRA page. I looked at the backlog view a little closer and found these possible options:

<img width="675" alt="screenshot 2018-12-03 at 10 02 15" src="https://user-images.githubusercontent.com/206108/49363649-89382500-f6e2-11e8-83d3-fff95bf67bff.png">

1. `$('[data-react-beautiful-dnd-draggable="0"]')` to find tickets using the fact that they are draggable and JIRA uses this react library. The downside is that maybe non-tickets are draggable too and JIRA may change how they implement drag&drop.
1. search for `BFQT-38`-style links (ticket links) and remove those ticket-links which seem to be non-tickets. ` $('a').toArray().filter((a) => a.href.match(/\/browse\/.+-\d+/))` (finds all links which go to `/browse/BFQT-38`-like URLS). This should be fairly stable since all ticket-summaries contain a link to the actual ticket
1. search for the little bug/story/epic image in ticket-summaries. It's always an image of the form
  ```
  <img class="obfuscated" role="presentation" src="https://org.atlassian.net/secure/viewavatar?size=xsmall&amp;avatarId=11111&amp;avatarType=issuetype">
  ```
  So we can search for images with `avatarType=issuetype` in the URL

**Comment by planktonic** [comment]
Yup, something like CSS-modules. Also means the classes will/could change on every deploy. Using the URL may be the most stable approach, as they really shouldn’t be breaking all their links any time soon?

**Comment by pmeinhardt** [comment]
This might be addressed by #137

**Comment by pmeinhardt** [comment]
Fixed via #137. We just need to release a new version soon-ish.
